python/lsst/ci/build.py

```diff
@@ -108,7 +108,8 @@ def report_result(self, retcode, logfile, fetched):
                 self.out.write("(already installed).\n")
                 self.out.flush()
             elif fetched:
-                self.out.write("(installed from binary).\n")
+                elapsed_time = time.time() - self.t0
+                print(f"(installed from binary) ({elapsed_time:.1f} sec).")
```

Comment:
```suggestion
                print(f"(installed from binary in {elapsed_time:.1f} sec).")
```
I think `(installed from binary in 5.2 sec).` reads a little better than 2 parentheticals.